### DISCUSSION TITLE
Overwrite: latest score will always overwrite previous scores (on global leaderboards) even if score is lower

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20004] [Ideas] [answered]
lets say you already have a 1m nm score on the leaderboard, but you want to replace that with a difficulty adjust score with cs7 or something, Overwrite would allow that score to be the one to show up on the leaderboard even though its half the value. (note: obviously the placement is based off the new score, you wont keep the #1 with a 500k score)
PP would not be overwritten, and the old score would still be available through mod leaderboards.

**Comment by smoogipoo** [comment, marked as answer]
We've discussed being able to preserve certain scores for the purpose of showing up in your profiles. This is probably an extension of that.

**Reply by peppy** [reply]
We were thinking of making it a post-fact thing, but honestly, implementing as a mod would be a pretty cool way of doing it (and greatly simplify the implementation). Not sure it'll be how we do it, but being able to see that someone was playing with the aim of overwriting is a cool bonus IMO.
